### ISSUE TITLE
Hardware requirements for Deepcell for training model on full HeLa_S3.npz dataset

### ISSUE THREAD

**Issue by manugarciaquismondo** [vanvalenlab/deepcell-tf#159]
Greetings,

I have deployed the Deepcell notebook example [_Interior-Edge Segmentation 2D Fully Convolutional.ipynb_](https://colab.research.google.com/drive/1dG08GtjVlCa771fzpErpyB1W80JjfW5s) on Google Colab. When I try to load the dataset _HeLa_S3.npz_ from Deepcell's AWS example bucket, the computing environment runs out of memory and crashes. Can you please advise on the recommended hardware requirements on which Deepcell has been proven to work for this notebook, so that I can set up a computing environment that meets these requirements and run Deepcell?

Thank you very much,  

@manugarciaquismondo and @cornhundred

**Comment by willgraf** [comment]
We ran these notebooks on our own NVIDIA DGX station, on one GPU with 16GB of memory.

The HeLaS3 dataset is almost 6 GB.  If you are unable to load the data into memory, we also have hosted the following 2 datasets which are both under 2GB:

[3T3_NIH.npz](https://s3-us-west-1.amazonaws.com/deepcell-data/nuclei/3T3_NIH.npz)

[HEK293.npz](https://s3-us-west-1.amazonaws.com/deepcell-data/nuclei/HEK293.npz)

---

These are the smallest datasets we have hosted currently.
